packages/superset-ui-legacy-preset-chart-big-number/src/BigNumber/transformProps.js

```diff
@@ -81,8 +84,16 @@ export default function transformProps(chartProps) {
   } else if (percentChange < 0) {
     className = 'negative';
   }
+  let metricFormat = yAxisFormat;
+  if (!yAxisFormat && chartProps.datasource && chartProps.datasource.metrics) {
```

Comment:
i think i'd prefer `!metricFormat` here

Reply:
`metricFormat` wasn't really necessary here... I made this one overwrite the passed in `yAxisFormat` like the others do. I think that's more straightforward.